pkgs/development/python-modules/gnuplot-kernel/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, writeText
+, notebook
+, metakernel
+, python
+, gnuplot
+, pytest
+}:
+
+let kernelSpecFile = writeText "kernel.json" (builtins.toJSON {
+       argv = [ "python" "-m" "gnuplot_kernel" "-f" "{connection_file}" ];
+       display_name = "gnuplot";
+       language = "gnuplot";
+       name = "gnuplot";
+    });
+in
+buildPythonPackage rec {
+  pname = "gnuplot_kernel";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "32ea9e8e52c8fd5f4c0611886cccc211f5c4038abfe89867f1ef4d6e2318268d";
+  };
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [ notebook metakernel gnuplot ];
```

Comment:
Should we not somewhere hardcode a path to gnuplot?

Reply:
Yes we should. Currently I am trying to get the `gnuplot_kernel` to work. It freezes on me for an unknown reason.